integrade/tests/api/v1/test_user_accounts.py

```diff
@@ -129,3 +129,22 @@ def test_change_password():
 
     response = client.post(urls.AUTH_TOKEN_DESTROY, {}, auth=auth)
     assert response.status_code == 204
+
+
+def test_cannot_delete_self():
+    """Ensure an user cannot delete their own account."""
```

Comment:
please update doc string with all the fields polarian likes :)

See example in test above. I just create the id with `uuidgen` on the command line.